docs/_templates/autosummary/class.rst

```diff
@@ -1,49 +1,43 @@
-{% if referencefile %}
+{%+ if referencefile %}
 .. include:: {{ referencefile }}
-{% endif %}
+{%+ endif -%}
```

Comment:
Heh, that's subtle I had to lookup that this strips whitespace in jinja2. For those curious see: https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control

Reply:
Yeah, this was just me being ridiculously particular about having the stubs generate looking "normal" with respect to whitespace.